<!doctype html>
<html lang="en">
<head>
    <meta charset="utf-8"/>
    <title>Global Biotic Interactions | About</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <style>
        #logos {
            height: 20%;
        }

        img {
            height: 100px;
            display: block;
            margin-left: auto;
            margin-right: auto;
        }
    </style>
    <link rel="stylesheet" href="http://code.jquery.com/ui/1.10.3/themes/smoothness/jquery-ui.css"/>
    <link rel="stylesheet" href="globi.css"/>
</head>
<body>
<div class="ui-widget">
    <h3>What is GloBI?</h3>

    <p>Global Biotic Interactions (GloBI) provides an infrastructure and data service that aggregates or combines
        existing biotic interaction datasets to provide easy access to biotic interaction data. Actually, some pages (<a
                href="references.html">here</a>, <a href="index.html">here</a> and <a href="browse/index.html">here</a>)
        of this <a href="https://github.com/globalbioticinteractions/globalbioticinteractions.github.io">website</a> are built on top of that
        infrastructure. The infrastructure software itself is available on <a
                href="https://github.com/jhpoelen/eol-globi-data">github</a> and the data is available in <a
                href="https://github.com/jhpoelen/eol-globi-data/wiki#accessing-species-interaction-data">various
            ways</a>. Like videos? Then you might like this <a href="http://vimeo.com/89373303">2 minute GloBI
            introduction video</a>.

    <h3>Why GloBI?</h3>

    <p><i>
      “The various books and journals of ornithology and entomology are like a row of beehives containing an immense amount of valuable honey, which has been stored up in separate cells by the bees that made it. The advantage, and at the same time the difficulty, of ecological work is that it attempts to provide conceptions which can link up into some complete scheme the colossal store of facts about natural history which has accumulated up to date in this rather haphazard manner. This applies with particular force to facts about the food habits of animals. Until more organised information about the subject is available, it is only possible to give a few instances of some of the more clear­cut niches which happen to have been worked out.”</i> - <a href="http://dx.doi.org/10.5962/bhl.title.7435">Charles Elton, 1927. <em>Animal Ecology</em>.</a></p>
    <p><i>“Biodiversity researchers have focused on diversity at the cost of ignoring the networks of interactions between organisms that characterize ecosystems.”</i> - <a href="http://dx.doi.org/10.1038/446029a">Kevin McCann,
        2007</a></p>

    <p><i>“An essential element of a truly inclusionary and democratic approach to science is to share data through
        publicly accessible data sets.”</i> - <a href="http://dx.doi.org/10.1093/biosci/biu169">Soranno et al., 2014</a>
    </p>

    <p>Now that folks have mapped the <a href="https://en.wikipedia.org/wiki/Human_Genome_Project">human genome</a>, put
        <a href="https://en.wikipedia.org/wiki/Neil_Armstrong">a man on the moon</a>, isn't it time to provide easy
        access to how, when and where organisms interact with each other so that we can better understand and better
        preserve our ecosystems? Perhaps GloBI can become the <a href="http://www.openstreetmap.org">OpenStreetMap</a>
        of ecology: a global map that shows how organisms rely on each other . . . </p>

    <p>By providing an infrastructure to capture and share interaction data, individual biologists can focus on
        gathering new interaction data and analyzing existing datasets without having to spend resources on (re-)
        building a cyberinfrastructure to do so.
    </p>

    <h3>Who makes GloBI possible?</h3>

    <p>GloBI is made possible by a community of software engineers, bioinformaticists and biologists. Software engineers
        such as <a href="http://linkedin.com/in/jhpoelen">Jorrit Poelen</a>, <a href="https://github.com/coding46">Göran
            Bodenschatz</a>, and <a href="https://github.com/reiz">Robert Reiz</a> collaborate with bioinformaticists
        like <a href="http://www.mendeley.com/profiles/chris-mungall/">Chris Mungall</a> and biologists like <a
                href="http://ccs.tamucc.edu/people/dr-james-d-simons/">Jim Simons</a>, <a
                href="http://ronininstitute.org/research-scholars/anne-thessen/">Anne Thessen</a>, and <a
                href="http://brianhayden.org">Brian Hayden</a> to capture, provide access to and use interaction data
        that is provided by <a href="/references.html">biologists and citizen scientists around the world</a>. GloBI is
        funded by EOL's <a href="http://eol.org/info/485">Rubenstein Fellowship Program</a>.</p>

    <h3>How to share data through GloBI?</h3>

    <p>GloBI exists because of people like yourself who share their interaction data, or refer to existing datasets that
        are not yet <a href="references.html">included in GloBI</a>. Do you have some interaction data you'd like to
        access through GloBI? Please read <a
                href="https://github.com/jhpoelen/eol-globi-data/wiki/How-to-Contribute-Data-to-Global-Biotic-Interactions%3F">
            this</a>.

    <h3>GloBI in the wild</h3>

    <p>The <a href="http://eol.org">Encyclopedia of Life</a> (see <a
            href="http://blog.globalbioticinteractions.org/2014/01/24/what-parasites-do-atlantic-croaker-host-find-it-on-the-encyclopedia-of-life/">blog
        post</a>) and <a href="http://gomexsi.tamucc.edu">Gulf of Mexico Species Interaction Database</a> (see <a
            href="http://blog.globalbioticinteractions.org/2013/09/17/want-to-explore-marine-foodwebs-in-the-gulf-of-mexico-visit-gomexsi/">blog
        post</a>) are currently using GloBI's data services. We also built some web apps on top of our data services: <a
            href="/references.html">list references and data sources</a>, <a href="/index.html">figure out who eats
        what</a> and <a href="/browse">browse interactions around the globe</a>.</p>

    <p>The <a href="http://r-project.org">R</a> community uses <a
            href="https://cran.r-project.org/web/packages/rglobi/index.html">rGloBI</a> (part of <a
            href="http://ropensci.org">rOpenSci</a>) to access interaction data (see <a
            href="http://timotheepoisot.fr/2015/05/10/hungry-caterpillars/">Hungry Caterpillars</a> for example). Also,
        students of <a href="http://iu.edu">Indiana University's</a> <a href="http://ivmooc.cns.iu.edu/">Information
            Visualization MOOC</a> created a <a href="http://dx.doi.org/10.6084/m9.figshare.1297762">Food-Web Map of the
            World</a> (class of 2014) and an <a href="http://danielabar.github.io/globi-proto">interactive ecosystem
            explorer</a> (class of 2015) (<a href="http://dx.doi.org/10.6084/m9.figshare.1414252">poster</a>, <a
                href="http://dx.doi.org/10.6084/m9.figshare.1414253">paper</a>) using GloBI data.</p>
    Some more mentions include <a
        href="http://www.bigbio.com.mx/bigbio/articulos/global-biotic-interactions-globi-bichos-vemos-relaciones-si-sabemos/">Global
    Biotic Interactions (GloBI). Bichos vemos, relaciones sí sabemos.</a></li>, <a
        href="http://aprendizajedistancia.blogspot.com/2015/05/datos-global-biotic-interactions.html">Datos: Global
        Biotic Interactions</a>, <a href="http://blog.openhelix.eu/?p=22003">OpenHelix's Video Tip of the Week</a>, <a
        href="http://nbviewer.ipython.org/github/Py4Life/TAU2015/blob/master/minilecture_API_globi.ipynb">Py4Life's
    mini-lecture using Python and GloBI</a> as part of <a href="http://py4life.github.io/TAU2015/">Python Programming
    for Biologist course at Tel-Aviv University</a> and <a href="https://github.com/terschure/reimagined-lamp">a colorful network visualization by Anneke ter Schure</a> (see <a href="https://rawgit.com/terschure/reimagined-lamp/master/BioticInteractions/index.html">live example</a>).
    <h3>How to cite GloBI?</h3>

    <p>Jorrit H. Poelen, James D. Simons and Chris J. Mungall. (2014). Global Biotic Interactions: An open
        infrastructure to share and analyze species-interaction datasets. <em>Ecological Informatics</em>. <a
                href="http://dx.doi.org/10.1016/j.ecoinf.2014.08.005">http://dx.doi.org/10.1016/j.ecoinf.2014.08.005</a>

    <h3 id="citations">Who cited GloBI?</h3>
    <ul>
      <li>Adams, J. et al., 2016. A century of Chinook salmon consumption by marine mammal predators in the Northeast Pacific Ocean. <em>Ecological Informatics</em>, 34, pp.44–51. Available at: <a href="http://dx.doi.org/10.1016/j.ecoinf.2016.04.010">http://dx.doi.org/10.1016/j.ecoinf.2016.04.010.</a></li>
      <li>Amano, T., Lamming, J.D.L. &amp; Sutherland, W.J., 2016. Spatial Gaps in Global Biodiversity Information and the Role of Citizen Science. <em>BioScience</em>, p.biw022. Available at: <a href="http://dx.doi.org/10.1093/biosci/biw022">http://dx.doi.org/10.1093/biosci/biw022</a>.
      </li>
      <li>Khabibullin, V.F., 2016. Location and foraging as basis for classification of biotic interactions. <em>Theory in Biosciences</em>. Available at: <a href="http://dx.doi.org/10.1007/s12064-016-0228-8">http://dx.doi.org/10.1007/s12064-016-0228-8</a>.</li>
    </ul>
    <ul>
      <li>Costello, M.J. et al., 2015. Biological and ecological traits of marine species. <em>PeerJ</em>, 3, p.e1201. Available at: <a href="http://dx.doi.org/10.7717/peerj.1201">http://dx.doi.org/10.7717/peerj.1201</a>.</li>
        <li>Foulger, R.E. et al., 2015. Representing virus-host interactions and other multi-organism processes in the
            Gene Ontology. <em>BMC Microbiol</em>, 15(1). Available at: <a
                    href="http://dx.doi.org/10.1186/s12866-015-0481-x">http://dx.doi.org/10.1186/s12866-015-0481-x</a>.
                  </li>
                  <li>Gray, C. et al., 2015. Joining the dots: An automated method for constructing food webs from compendia of published interactions. <em>Food Webs</em>, 5, pp.11–20. Available at: <a href="http://dx.doi.org/10.1016/j.fooweb.2015.09.001">http://dx.doi.org/10.1016/j.fooweb.2015.09.001</a>.</li>
        <li>Hortal, J. et al., 2015. Seven Shortfalls that Beset Large-Scale Knowledge of Biodiversity. <em>Annual Review of Ecology, Evolution, and Systematics</em>, 46(1). Available at: <a href="http://dx.doi.org/10.1146/annurev-ecolsys-112414-054400">http://dx.doi.org/10.1146/annurev-ecolsys-112414-054400i</a>.</li>
        <li>Kissling, W.D. et al., 2015. Towards global interoperability for supporting biodiversity research on
            essential biodiversity variables (EBVs). <em>Biodiversity</em>, pp.1–9. Available at:
            <a href="http://dx.doi.org/10.1080/14888386.2015.1068709">http://dx.doi.org/10.1080/14888386.2015.1068709</a>.
        </li>
        <li>Poisot, T. et al., 2015. mangal - making ecological network analysis simple. <em>Ecography</em>. Available
            at: <a href="http://dx.doi.org/10.1111/ecog.00976">http://dx.doi.org/10.1111/ecog.00976</a>.
        </li>
        <li>Poisot, T. et al., 2015. Synthetic datasets and community tools for the rapid testing of ecological hypotheses. <em>Ecography</em>. Available at: <a href="http://dx.doi.org/10.1111/ecog.01941">http://dx.doi.org/10.1111/ecog.01941</a>.</li>
        <li>Soranno, P.A. et al., 2015. Building a multi-scaled geospatial temporal ecology database from disparate data
            sources: fostering open science and data reuse. <em>GigaSci</em>, 4(1). Available at: <a
                    href="http://dx.doi.org/10.1186/s13742-015-0067-4">http://dx.doi.org/10.1186/s13742-015-0067-4</a>.
        </li>
        <li>Szoboszlai, A.I. et al., 2015. Forage species in predator diets: Synthesis of data from the California
            Current. <em>Ecological Informatics</em>. Available at: <a
                    href="http://dx.doi.org/10.1016/j.ecoinf.2015.07.003">http://dx.doi.org/10.1016/j.ecoinf.2015.07.003</a>.
        </li>
        <li>Thessen, A.E. et al., 2015. Data Infrastructures for Estuarine and Coastal Ecological Syntheses. <em>Estuaries and Coasts</em>. Available at: <a href="http://dx.doi.org/10.1007/s12237-015-0045-1">http://dx.doi.org/10.1007/s12237-015-0045-1.</a></li>
        <li>Thessen, A.E. et al., 2015. Emerging semantics to link phenotype and environment. <em>PeerJ</em>, 3, p.e1470. Available at: <a href="http://dx.doi.org/10.7717/peerj.1470">http://dx.doi.org/10.7717/peerj.1470</a>.</li>
    </ul>
</div>
<br/><br/>
<table>
    <tr>
        <td width="60%">
            <a href="http://eol.org"><img src="img/EOL_logo_globeHIGHRES.jpg"></img></a>
        </td>
        <td><a href="http://gomexsi.tamucc.edu"><img src="img/gomexsi-logo.png"></img></a>
        </td>
    </tr>
</table>
<script src="menu.js" charset="utf-8"></script>
<script>
    buildMenu();
</script>
</body>
</html>
